formats/code/index.html

```diff
@@ -0,0 +1,112 @@
+---
+layout: reference
+---
+<div class="box">
+  <h2>Introduction to the Git Reference</h2>
+  <div class="block">
+    <p>
+    This is the Git reference site. It is meant to be a quick
```

Comment:
Dit kan dan weer verwarrend werken

Reply:
Wat is de reden voor het toevoegen van dit bestand? Is dit een voorbeeld voor code markup?

Reply:
Inderdaad, iets van een blok code markup. Deze is toevallig free to use zonder daar een clause voor toe te moeten voegen en ziet er wel mooi uit. Denk je dat het beter is anders zelf een blokje te schrijven?

Reply:
Het is veel beter als we (of desnoods marketing) hier een mooie pagina voor maakt. De vraag is ook of html de beste optie is, of code. Wie is er aan het evalueren en wordt hier blij van? Is het beter om (ook) clojure / java / etc bestanden toe te voegen?

Reply:
Prima, zal ik deze sectie dan verwijderen?

Reply:
Of veranderen door inhoud die over Git for Confluence gaat, of een compleet random html bestand, of beter iets van Java code ofzo?

Reply:
I'll look for something!